prueba.py

```diff
@@ -0,0 +1,10 @@
+import optparse
```

Comment:
Why is this file named `prueba.py` in spanish, as far as we know, we are programing in English language.